mlflow/R/mlflow/R/tracking-rest.R

```diff
@@ -57,8 +69,25 @@ mlflow_rest <- function( ..., client, query = NULL, data = NULL, verb = "GET", v
     ),
     stop("Verb '", verb, "' is unsupported.")
   )
-  if (identical(response$status_code, 500L)) {
-    stop(xml2::as_list(content(response))$html$body$p[[1]])
+  if (response$status_code != 200) {
+    message_body <- tryCatch(
+      paste(content(response, "parsed", type = "application/json"), collapse = "; "),
+      error = function(...) {
```

Comment:
Usually we use `function(cnd)` or `function(e)` etc. for the signature for handling conditions, since it's a condition object that's passed to the function.